MANIFEST.in

```diff
@@ -0,0 +1 @@
+include version.h
```

Comment:
What is this file needed for?  Also it is missing a newline which makes git cranky